tests/test_packages/test_skills/test_simple_data_request/test_handlers.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the tests of the handler classes of the simple_data_request skill."""
+
+import logging
+from pathlib import Path
+from typing import cast
+from unittest.mock import patch
+
+import pytest
+
+from aea.protocols.dialogue.base import DialogueMessage
+from aea.test_tools.test_skill import BaseSkillTestCase
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.skills.simple_data_request.dialogues import HttpDialogues
+from packages.fetchai.skills.simple_data_request.handlers import HttpHandler
+
+from tests.conftest import ROOT_DIR
+
+
+class TestHttpHandler(BaseSkillTestCase):
```

Comment:
You Could do the setup once by defining an intermediate class `SimpleDataRequestTestCase(BaseSkillTestCase)` which implements the setup method and then reuse it

Reply:
Good suggestion. 

Done.